edweiss-app/components/quiz/QuizComponents.tsx

```diff
@@ -101,48 +130,50 @@ export const TFDisplay: ReactComponent<{ exercise: Quizzes.TF, selected: boolean
 	}
 
 	return (
-		<TView mb={"xs"} bb={1} borderColor='surface0' m={"md"} radius={'lg'} p={"md"} pb={"xl"}>
-
-			<TView mb={"md"} radius={"xl"} p={"md"}>
-				<RichText size={"lg"}>
+		<TView bb={disableBottomBar ? 0 : 1} borderColor='surface0' mx={12} radius={'lg'} pb={"md"}>
+			<TView p={"sm"}>
+				<TText>
 					{exercise.question}
-				</RichText>
+				</TText>
 			</TView>
 
-			<RadioSelectables data={[trueSelectable, falseSelectable]} onSelection={(value) => {
-
-				handleSelection(value)
-			}} value={selected} />
-
+			<TView ml={'sm'}>
+				<RadioSelectables data={[trueSelectable, falseSelectable]} onSelection={(value) => {
+					handleSelection(value)
+				}} value={selected} />
+			</TView>
 		</TView>
 
 	);
 });
 
 
-export const TFResultDisplay: ReactComponent<{ exercise: Quizzes.TF, selected: boolean | undefined, result: boolean; }> = ({ exercise, selected, result }) => {
+export const TFResultDisplay: ReactComponent<{ exercise: Quizzes.TF, selected: boolean | undefined, result: boolean, disableBottomBar?: boolean }> = ({ exercise, selected, result, disableBottomBar }) => {
+	const trueSelectable: RadioSelectable<boolean> = {
+		label: t('quiz:quiz_display.true'),
+		value: true,
+		color: checkResultColor(checkTFCorrect(selected, true, result))
+	}
+	const falseSelectable: RadioSelectable<boolean> = {
+		label: t('quiz:quiz_display.false'),
+		value: false,
+		color: checkResultColor(checkTFCorrect(selected, false, result))
+	}
```

Comment:
You can merge it in on component taking a boolean as parameter 
```suggestion
	const trueSelectable(bool: boolean): RadioSelectable<boolean> = {
		label: t(`quiz:quiz_display.${bool}`),
		value: bool,
		color: checkResultColor(checkTFCorrect(selected, bool, result))
	}
```